### PR TITLE
Add in Reference from #63 for modern data workflow for living data

### LINKED ISSUE #63: Developing a modern data workflow for living data
https://doi.org/10.1101/344804

Should we include this in our discussion?   Continuous integration for data management has parallels to continuous integration for manuscripts.

### PR DESCRIPTION
**Pull request by vsmalladi**
Closes #63
_agitter: edited to close related issue upon merge_

### CODE REVIEW

**Review by agitter — COMMENTED** [review]
This placement of the reference suggests that the paper describes benefits of the open workflow to the general non-scientist public. Is that the main focus, or is it aimed toward the scientific community?

Before merging, we should also add this reference to the continuous integration paragraph.  I think it fits better there than the "paper of the future" paragraph.

**Comment by vsmalladi** [comment]
Yes, I think I should move the reference to the 'scientific peers' and will add the reference to the continuous integration section.

**Review by agitter — APPROVED** [review]
This placement looks okay to me.  We could pull the reference out separately later to describe what this approach does in more detail, but it may be slightly out of scope.

Let's wait for @dhimmel to review before merging.

**Comment by vsmalladi** [comment]
Sounds good.

File: content/02.main-text.md
Comment (dhimmel) [inline content/02.main-text.md]: I'm in favor of removing the reference to "Developing a modern data workflow for living data" here. It's a bizarrely specific reference when we don't cite other more general works. We probably should cite more general works, but these points are also rather self evident.

Comment (agitter) [reply content/02.main-text.md]: @dhimmel despite what I wrote above, upon further consideration I agree with you about removing it here.  It fits better in the continuous integration paragraph.  Are you okay adding it in that paragraph?

Comment (dhimmel) [reply content/02.main-text.md]: Yeah I think it is appropriate for the CI sentence... It's a similar use case of CI

**Comment by agitter** [comment]
@vsmalladi will you be able to make this small change soon so we can merge?  @dhimmel's suggestion above will also resolve the merge conflict.

**Comment by vsmalladi** [comment]
@agitter Yes I will be able to edit it now.

File: content/02.main-text.md
Comment (dhimmel) [inline content/02.main-text.md]: Has been added twice.

**Review by dhimmel — CHANGES_REQUESTED** [review]

File: content/02.main-text.md
Comment (vsmalladi) [reply content/02.main-text.md]: @dhimmel Ah thanks for the catch. Will fix now.

**Review by dhimmel — APPROVED** [review]

**Pull request merged**
